couch/tests/test_couchv2.py

```diff
@@ -214,81 +214,51 @@ def test_check_without_names(aggregator, gauges):
 
 @pytest.mark.usefixtures('dd_environment')
 @pytest.mark.integration
-def test_only_max_nodes_are_scanned(aggregator, gauges):
+@pytest.mark.parametrize('number_nodes', [1, 2, 3])
+def test_only_max_nodes_are_scanned(aggregator, gauges, number_nodes):
```

Comment:
Note: I simplified some tests to only assert what's necessary:

- `test_only_max_nodes_are_scanned`: assert that that only tags with a limited number of nodes are submitted
- `test_only_max_dbs_are_scanned`:  assert that that only tags with a limited number of dbs are submitted